frontend/src/scenes/paths/Paths.js

```diff
@@ -233,18 +233,16 @@ export function Paths() {
             {loadedFilter.path_type === AUTOCAPTURE && (
                 <div style={{ margin: 10 }}>Click on a tag to see related DOM tree</div>
             )}
-            <div ref={canvas} className="paths" style={{ height: '90vh' }} data-attr="paths-viz">
-                {!pathsLoading && paths && paths.nodes.length === 0 ? (
-                    <NoData />
-                ) : (
-                    pathsLoading && (
-                        <div className="loading-overlay mt-5">
-                            <div />
-                            <Loading />
-                            <br />
-                        </div>
-                    )
-                )}
+            <div
+                style={{
+                    minHeight: '70vh',
+                    position: 'relative',
+                }}
+            >
```

Comment:
I just matched this with the trends graph style. There might be some interference I'm not thinking about